sdk/rms_sdk/ModernAPI/ModernAPI.pro

```diff
@@ -61,10 +67,10 @@ HEADERS += \
     ProtectedFileStream.h \
     CustomProtectedStream.h \
     HttpHelper.h \
-    ModernAPIExport.h \
```

Comment:
do you need to add here the api_exports.h?

Reply:
No. api_exports.h is a part of the 'Profile' folder.